src/Nancy.ViewEngines.Razor/VisualBasic/VisualBasicClrTypeResolver.cs

```diff
@@ -0,0 +1,85 @@
+namespace Nancy.ViewEngines.Razor.VisualBasic
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Web.Razor.Tokenizer.Symbols;
+
+    internal class VisualBasicClrTypeResolver : ClrTypeResolver<VBSymbolType, VBSymbol>

```

Comment:
Most code here is a duplicate of the one for C-Sharp. Should be possible to create a shared base class and only expose methods to override / implement for the differences